src/components/date-time.vue

```diff
@@ -16,22 +16,33 @@ except according to the terms contained in the LICENSE file.
 <script>
 import { DateTime } from 'luxon';
 
-import { formatDate } from '../util/date-time';
+import { formatDateTime } from '../util/date-time';
 
 export default {
   name: 'DateTime',
   props: {
     iso: String, // eslint-disable-line vue/require-default-prop
-    blank: String // eslint-disable-line vue/require-default-prop
+    blank: {
+      type: String,
+      default: ''
+    }
   },
   computed: {
+    fromISO() {
+      return DateTime.fromISO(this.iso);
+    },
+    withLocale() {
+      return this.fromISO.locale === this.$i18n.locale
+        ? this.fromISO
+        : this.fromISO.setLocale(this.$i18n.locale);
```

Comment:
Does `setLocale` not already do this check and do nothing if the specified locale is the same? Not a big deal but I was surprised to see this logic here.

Reply:
`setLocale()` seems to return a new `DateTime` object even if the locale is the same, so I added the ternary so that `this.fromISO` can be reused if the locale is the same.

Reply:
Fair enough!